src/WizardStep.php

```diff
@@ -180,4 +180,22 @@ protected function rules(): array
             ->mapWithKeys(fn (Field $field) => [$field->name => $field->rules])
             ->all();
     }
+
+    /**
+     * The custom validation messages
+     *
+     * @return array<string, string>
+     */
+    protected function messages(): array {
+        return [];
+    }
+
+    /**
+     * The custom validation messages
```

Comment:
This description looks like a copy&paste mistake :)

Reply:
Fixed :)